src/components/layout/MobileFooter/MobileFooter.scss

```diff
@@ -9,11 +9,20 @@
   justify-content: space-between;
   border-bottom: 0.5px solid var(--light-overlay-015);
   border-top: 0.5px solid var(--light-overlay-015);
-  height: 48px;
+  height: auto;
+  padding: 0.5rem 1rem;
+  padding-bottom: calc(env(safe-area-inset-bottom) + 0.5rem);
 
   &__link {
+    display: flex;
+    align-items: center;
+    padding: 0.5rem 2rem;
+
     svg {
-      width: 1.5rem;
+      width: 24px;
+      min-width: 24px;
+      height: 24px;
+      min-height: 24px;
```

Comment:
why switch to `px` units here?

Reply:
Oh, had a really weird issue on the mobile footer; when we switch between tabs, the icons are shifting. I was trying to find the cause, and needed to make sure their sizing, I think it's not what I thought, so I'll be reverting those, good catch